src/index.ts

```diff
@@ -13,20 +13,22 @@ import {
 	parseTsconfig,
 	createPathsMatcher,
 	createFilesMatcher,
+	TsConfigResult,
 } from 'get-tsconfig';
 import type { TransformOptions } from 'esbuild';
 
 const isPathPattern = /^\.{0,2}\//;
 const isTsFilePatten = /\.[cm]?tsx?$/;
 const nodeModulesPath = `${path.sep}node_modules${path.sep}`;
 
+const tsconfigCache = new Map<string, TsConfigResult>();
 const tsconfig = (
 	process.env.ESBK_TSCONFIG_PATH
 		? {
 			path: path.resolve(process.env.ESBK_TSCONFIG_PATH),
 			config: parseTsconfig(process.env.ESBK_TSCONFIG_PATH),
```

Comment:
Here too:
https://github.com/privatenumber/get-tsconfig#parsetsconfigtsconfigpath-cache

Reply:
done, aslo repeat it in esm-loader